packages/create-pantheon-decoupled-kit/src/templates/next-drupal/scripts/get-locales.js

```diff
@@ -0,0 +1,22 @@
+const fetch = require('isomorphic-fetch');
+
+const getLocales = async () => {
+	try {
+		const res = await fetch(
+			`${process.env.BACKEND_URL}/jsonapi/configurable_language/configurable_language`,
+		);
+
+		if (res.status === 200) {
+			return ['en', 'es'];
+		} else {
+			return ['en'];
+		}
+	} catch (error) {
```

Comment:
When I run create from outside of the monorepo using NPM this code block actually seems to be executing somehow...

```
found 0 vulnerabilities
Linting...

> next-drupal-starter@0.1.0 lint
> next lint

There was an error fetching language data. Only absolute URLs are supported
>> Enure BACKEND_URL is set in .env.development.local
Defaulting locale to ['en']
✔ No ESLint warnings or errors
addWithDiff success
runInstall success
runLint success
```

Reply:
This is expected since `next lint` executes the `next.config.js`.